configuration/config-txt/video.md

```diff
@@ -483,27 +483,31 @@ Setting `hdmi_ignore_hotplug` to `1` pretends that the HDMI hotplug signal is no
 
 ### disable_overscan
 
-Set `disable_overscan` to `1` to disable [overscan](../raspi-config.md#overscan).
+Set `disable_overscan` to `1` to disable [overscan](../raspi-config.md#overscan). The default value is `0`.
+
+Disabling overscan is accomplished by setting `overscan_left`, `overscan_right`, `overscan_top` and `overscan_bottom` together to `0`. Any further additional overscan setting such as `overscan_scale` can still be applied.
 
```

Comment:
I think this should say:
Set `disable_overscan` to `1` to disable the default values of overscan the firmware sets.
The default values are `48` for HD CEA modes, `32` for SD CEA modes and `0` for DMT modes.

Reply:
I'm not sure that is precise enough. What will be the values if you *disable the default values*?
I understand these values are set to (or left as?) `0`, hence why I detailed precisely what it does.

Also mentioning the default values of other options is not appropiate here IMHO.
Likewise in the other options of the documentation, each option precisely defines what is its default value independently and self-contained. This enables users to quickly consult the documentation as a reference guide, without needing to jump back and forth other sections.